haskell-src/Concordium/ID/Types.hs

```diff
@@ -271,10 +274,45 @@ instance ToJSONKey IdentityProviderIdentity where
 -- Account signatures (eddsa key)
 type AccountSignature = Signature
 
--- encryption key for accounts (Elgamal?)
+-- |An Elgamal encryption key. Note that this is a full key with a specific
+-- generator @g@ and the @g^s@ for a secret key @s@. In our use the generator is
+-- in fact always the same (it is part of the shared cryptographic parameters of
+-- the chain, set in genesis), so we could put it into a shared context instead
+-- to save some space.
 newtype AccountEncryptionKey = AccountEncryptionKey {_elgamalPublicKey :: ElgamalPublicKey}
     deriving (Eq, Show, Serialize, FromJSON, ToJSON) via ElgamalPublicKey
 
+-- |A marker type that has the 'FBS.FixedLength' instance to determine the size
+-- of the account encryption key in bytes.
+data AccountEncryptionKeySize
+
+instance FBS.FixedLength AccountEncryptionKeySize where
+  fixedLength _ = 2 * 48
```

Comment:
If this 48 number is the same 48 number as the credential ID size, make it explicitly so: 

```
fixedLength _ = 2 * fixedLength (undefined :: RegIdSize)
```

If it's not, please document where does it come from.

Reply:
It is not. I mean it is, but only because the same group is used for both. But conceptually they are not to be conflated